internal/postprocess/dependencyTrack.go

```diff
@@ -131,7 +136,27 @@ func (d *DependencyTrackPostProcess) PostProcess(message internal.LagoonInsights
 		return err
 	}
 
-	client, err := dtrack.NewClient(d.ApiEndpoint, dtrack.WithAPIKey(d.ApiKey))
+	// Support Lagoon API env var override.
+	apiEndpoint := d.ApiEndpoint
+	apiKey := d.ApiKey
+	if message.Payload["dependency_track_endpoint"] != "" {
```

Comment:
So this logic makes sense - although we will discuss this in our technical directions. I wonder if what we might do here is turn this into a separate post-processor. That way it's not an override, but an additional functionality (so, if admins wanted to keep a list of SBOMS at a cluster level, they could do that, and individuals could additionally be able to ship them to their own DT).

Refactoring this into two post processors should be trivial, and it opens up a ton of cool configuration options for us.

Thoughts?